src/Extensions/TimedScopes/ActivityExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+
+namespace Microsoft.Omex.Extensions.TimedScopes
+{
+	/// <summary>
+	/// Extensions for Activity
+	/// </summary>
+	public static class ActivityExtensions
+	{
+		/// <summary>
+		/// Get user hash from activity
+		/// </summary>
```

Comment:
Missing `<params>` comments. Applies to a number of places throughout

Reply:
https://github.com/microsoft/Omex/pull/153#discussion_r391957268